paket.dependencies

```diff
@@ -10,6 +10,7 @@ nuget FSharpLint.Core prerelease
 nuget VSSDK.IDE.Deploy.14
 nuget FSharp.Management
 nuget FsPickler
+nuget FSharp.Interop.Dynamic
```

Comment:
Possibly, I searched for how to do dynamic in F#, and `FSharp.Interop.Dynamic` was the first suggestion :)

Anyway, I'd be happy to replace it with plain old reflection, unfortunately I can't yet figure out how to do this in F#... Is there an example of how this function is used?